build.sbt

```diff
@@ -78,6 +68,20 @@ lazy val natchezSlf4j = project
     )
   )
 
+val silencerVersion = "1.4.4"
+lazy val natchezDoobie = project
+  .in(file("natchez-doobie"))
+  .settings(common :+ (name := "natchez-doobie"))
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.tpolecat" %% "natchez-core" % "0.0.8",
+      "org.tpolecat" %% "doobie-core"  % "0.8.4",
+      "org.tpolecat" %% "doobie-h2"    % "0.8.4",
+      "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full,
+      compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
```

Comment:
how come this one? forcing no warnings at all

Reply:
I use it to silence the deprecation warning that get because one of the JDBC `PreparedStatement` functions we pass through is deprecated

Reply:
i think it is only used here https://github.com/ovotech/effect-utils/pull/21/files#diff-d392b59d1414737fc3b433e92fd22dc2R15 not in general